client/src/components/Nav/index.js

```diff
@@ -18,14 +17,13 @@ function Navigation() {
           <Nav className="mr-auto">
             <Nav.Link href="/">Home</Nav.Link>
             <Nav.Link href="#restaurants">Restaurants</Nav.Link>
```

Comment:
@Kina-rain This component is updated on master - there is a link to the spinner here now instead of the restaurants - can you make sure you do a git pull/merge w master on your branch and then push up again?